packages/pn-commons/src/components/__test__/CustomDatePicker.test.tsx

```diff
@@ -26,6 +26,16 @@ describe('test CustomDatePicker component', () => {
   });
 });
 
+describe('test CustomDatePicker component in mobile', () => {
+  it('renders the component with mobile window', () => {
+    window.matchMedia = createMatchMedia(800);
+    const { getByPlaceholderText, container } = render(<RenderDatePicker />);
```

Comment:
In this test we are not testing if mobile view is shown, because we are olny testing that an input is shown without testing the popup.

Furthermore, there is no test that tests the desktop view

Reply:
this wasn't done